test-monitor/nitestmonitor.yml

```diff
@@ -0,0 +1,1277 @@
+        type: string
+        required: true
+    get:
+      operationId: RootEndpointWithVersion
+      x-ni-request-all-privileges: true
+      summary: API version information
+      description: Returns available operations for a single version of the API.
+      tags: [versioning]
+      security: []
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/V1Operations'
+        404:
+          description: Not Found
+          schema:
+            $ref: '#/definitions/Error'
+  /v1/query-results:
+    post:
+      operationId: query-results
+      x-ni-operation: queryResults
+      x-ni-privilege: Read
+      summary: Queries test results
+      description: Queries test results
+      tags:
+        - results
+      parameters:
+        - in: body
+          name: postBody
+          description: A result query object
+          required: false
+          schema:
+            $ref: '#/definitions/TestResultQueryObject'
+        - in: query
+          name: skip
+          type: integer
+          default: 0
+        - in: query
+          name: take
+          type: integer
+          default: -1
+      responses:
+        200:
+          $ref: '#/responses/ResultsQueryResponse'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/results:
+    put:
+      operationId: update-results
+      x-ni-operation: updateResults
+      x-ni-privilege: Write
+      summary: Update test results
+      description: Update test results
+      tags:
+        - results
+      parameters:
+        - in: body
+          name: requestBody
+          description: A list of test results to update
+          required: true
+          schema:
+            description: A list of test results to update
+            title: UpdateTestResultsRequest
+            type: object
+            required:
+              - results
+            properties:
+              results:
+                description: An array of test results to update
+                type: array
+                items:
+                  $ref: '#/definitions/TestResultUpdateRequestObject'
+              replace:
+                description: Replace the existing fields instead of merging them
+                type: boolean
+                default: false
+              determineStatusFromSteps:
+                description: Determine test result status from the test step statuses
+                type: boolean
+                default: false
+      responses:
+        200:
+          $ref: '#/responses/ResultsPartialSuccessResponse'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+    post:
+      operationId: create-results
+      x-ni-operation: createResults
+      x-ni-privilege: Write
+      summary: Create test results
+      description: Create test results
```

Comment:
Creates new test results

(or cut the summary)
(this note applies to all the summaries in this section)